docker/rpms/Fedora_34/Fedora34.dockerfile

```diff
@@ -1,6 +1,6 @@
 # vim: ft=dockerfile
 
-FROM fedora:33
+FROM fedora:34
```

Comment:
I see a rather strange timestamp in the buildlog:
https://github.com/cobbler/cobbler/runs/2181937721#step:3:9351

Reply:
Since this is only CI, I would not consider this important but I will open an issue for investigation for it.